src/main/java/org/example/repository/StudentFeeRepository.java

```diff
@@ -25,7 +29,42 @@ public static StudentFeeRepository getInstance() {
     // 레포에 들어있는 csv 파일을 읽어와서 StudentInfo 객체를 만들고 HashMap에 저장하는 메소드
     // 필요할 경우 private method를 추가하여 구현
     public void initStudentInfo() {
-
+        try {
+            // 엑셀 파일 읽기
+            FileInputStream fis = new FileInputStream("23.xlsx");
+            Workbook workbook = WorkbookFactory.create(fis);
+            // 첫번째 시트(하나밖에 없으므로)
+            Sheet sheet = workbook.getSheetAt(0);
+            // 각 행(학생 정보)
+            for (Row row : sheet) {
+                // 첫번째 셀과 마지막 셀 건너뜀
+                if (row.getRowNum() == 0 || row.getRowNum() == sheet.getLastRowNum()-1 || row.getRowNum() == sheet.getLastRowNum()) {
+                    continue;
+                }
+                else {
+                    // 각 셀의 값(학번, 이름, 학생회비 납부) 가져오기
+                    String studentId = row.getCell(1).toString();
+                    String studentName = row.getCell(2).toString();
+                    String studentIsPayedStudentFee = row.getCell(4).toString();
+                    boolean isPayed = false;
+                    if (studentIsPayedStudentFee.equals("o")) {
+                        isPayed = true;
+                    }
```

Comment:
```suggestion
                    boolean isPayed = studentIsPayedStudentFee.equals("o");
```

이렇게 작성해도 됩니다. 그런데 o 만 처리할지는 고민해봐야겠네요

Reply:
훨씬 깔끔하네요!